### PR TITLE
Add Clojure Documentation.

### PR DESCRIPTION
**Pull request by dharrigan**
Hi @bruno-garcia 

Hope this is okay. Let me know. I don't have a lot of time to write up a *complete* set of documentation, as found under the Java project, but since the Clojure library simply wraps the Java library, then everything discussed there is applicable here (with the normal Clojure cavets :-) ) I think the github docs (linked in the documentation) is all that is required for now.

-=david=-

### CODE REVIEW

File: src/wizard/clojure/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Configure
+
+Initialize Sentry with the `init!` function. Remember to configure the DSN and any other [settings](https://github.com/getsentry/sentry-clj) you need:
+
+```clojure
+(ns foo
+ (:require
+  [sentry-clj.core :as sentry]))
+
+(def dsn "https://abcdefg@localhost:9000/2")
```
Comment (bruno-garcia) [inline src/wizard/clojure/index.md:37]:
```suggestion
(def dsn "__PUBLIC_DSN__")
```

This gets replaced with the users DSN for the project they just created

File: src/wizard/clojure/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+name: Clojure
+doc_link: https://docs.sentry.io/platforms/clojure/
```
Comment (bruno-garcia) [inline src/wizard/clojure/index.md:3]: We should add some docs first before we merge this. Right now it 404's

Comment (bruno-garcia) [reply src/wizard/clojure/index.md:3]: We'd need a `clojure` folder here with at least an `index.mdx`:

https://github.com/getsentry/sentry-docs/tree/master/src/platforms

**Comment by bruno-garcia** [comment]
Reminder we need to figure out docs for clojure

**Comment by dharrigan** [comment]
Hi Bruno,

The docs are autogenerated by `cljdocs` and available here: https://cljdoc.org/d/io.sentry/sentry-clj/3.1.134/doc/readme

Do you want to simply link to them?

-=david=-

**Comment by bruno-garcia** [comment]
Oh good to know. API docs we usually host on github pages of the repo, or link to it from the repository (I just added the link you gave to the repo).

But here would be nice to have the complete docs.

We have docs on how to contribute to the docs, on how to add a platform:

https://docs.sentry.io/contributing/approach/write-getting-started/

Is this something you'd be interested in contributing?

We can inherit the content from Java, all that's needed are the code snippets in Clojure instead of Java.

**Comment by dharrigan** [comment]
I'll pick this up.

**Comment by bruno-garcia** [comment]
Thanks @dharrigan, let me know if you need anything.

**Comment by bruno-garcia** [comment]
@dharrigan Just realized you might want to take a look at this:
https://docs.sentry.io/contributing/approach/write-getting-started/

Basically splitting things in 3 steps. I just did it for #2859 yesterday although it's broken there and need guidance to move forward. But I think Clojure will be very similar where it's we're adding the initial `/platform/name` docs and also a wizard.

**Comment by bruno-garcia** [comment]
I'm experimenting with some changes here: https://github.com/getsentry/sentry-docs/pull/2913

Once the build passes, and the preview on vercel makes sense, maybe you can pull that branch into yours and we merge?

**Pull request closed without merging**